lib/si-events-rs/src/audit_log/v2.rs

```diff
@@ -1,7 +1,10 @@
 use serde::{Deserialize, Serialize};
 use strum::Display;
 
-use crate::{Actor, AttributeValueId, ChangeSetId, ComponentId, PropId, SchemaVariantId, SecretId};
+use crate::{
+    ActionKind, ActionPrototypeId, Actor, AttributeValueId, ChangeSetId, ComponentId, FuncId,
+    PropId, SchemaVariantId, SecretId,
+};
 
 #[derive(Debug, Deserialize, Serialize, Clone, PartialEq, Eq)]
 pub struct AuditLogV2 {
```

Comment:
Calling this out for other reviewers: we are adding variants to the end of an enum that is serialized and deserialized with `serde_json`, so this is safe for backwards compatibility. We do not use a special form of serialization here: just the standard one.